core/services/pipeline/orm.go

```diff
@@ -278,6 +310,16 @@ func (o *orm) processNextUnclaimedTaskRun(ctx context.Context, fn ProcessTaskRun
 			if err != nil {
 				return errors.Wrap(err, "could not notify pipeline_run_completed")
 			}
+
+			elapsed := time.Since(ptRun.CreatedAt)
+			promPipelineRunTotalTimeToCompletion.WithLabelValues(pipelineSpecIDStr).Set(float64(elapsed))
+			if finalErrors, is := result.Error.(FinalErrors); is && finalErrors.HasErrors() {
```

Comment:
these are the same line `promPipelineRunErrors.WithLabelValues(pipelineSpecIDStr).Inc()` - do we just want to add an `|| result.Error != nil`?